build_projects/dotnet-cli-build/DebTargets.cs

```diff
@@ -13,14 +13,14 @@ namespace Microsoft.DotNet.Cli.Build
     public class DebTargets
     {
         [Target(nameof(GenerateSdkDeb))]
-        [BuildPlatforms(BuildPlatform.Ubuntu)]
+        [BuildPlatforms(BuildPlatform.Ubuntu, "14.04")]
```

Comment:
Bryan will know for sure if we even can yet.  For some reason I thought there were also issues around being able to provide a 16.04 specific feed in VSO.

Note we already have debt here for Debian proper.